ui/src/app/settings/selectors/config.js

```diff
@@ -131,4 +131,24 @@ config.proxyType = createSelector(
   }
 );
 
+/**
+ * Returns the MAAS config for MAAS name.
+ * @param {Object} - The redux state.
+ * @returns {String} Then MAAS name.
+ */
+config.maasName = createSelector(
+  [config.all],
+  configs => getValueFromName(configs, "maas_name")
+);
+
+/**
+ * Returns the MAAS config for enable analytics.
+ * @param {Object} - The redux state.
+ * @returns {Boolean} Enable analytics.
+ */
+config.enableAnalytics = createSelector(
```

Comment:
`config.analyticsEnabled` might be better, `config.enableAnalytics` sounds like a function rather than a property.

Reply:
done